lib/ace/ext/papeeria/spellchecker.coffee

```diff
@@ -1,39 +1,119 @@
 define((require, exports, module) ->
     # This will be removed and replaced with real data from server
     # BTW, this is kinda template for data to send from server, eh?
-    TEST_JSON_TYPOS = [
-        {
-            word: "blablabla",
-            corrections: [
-                "blah-blah",
-                "blabber",
-                "blabbed",
-                "salable"
-            ]
-        }
-    ]
-
-
-    # SpellChecker is a class that by now provides highlighting of words that takes from JSON.
-    class SpellChecker
+    #
+    # Note: I modified the structure of json, because I thought, maybe
+    # json mapping would be faster than searching for token in a loop?
+    TEST_JSON_TYPOS = {
+        "blablabla": [
+            "blah-blah",
+            "blabber",
+            "blabbed",
+            "salable"
+        ]
+    }
+
+    # Class by now provides highlighting of words that takes from JSON.
+    exports.SpellChecker = class SpellChecker
```

Comment:
Can't say that I like this assignment. Pls leave class definition and export assignment separate.